students/itsabhi/Lesson3/slicing.py

```diff
@@ -0,0 +1,47 @@
+def exchange_first_last(a):
+    a_seq = a
+    a_new_seq = a_seq[-1:] + a_seq[1:-1] + a_seq[0:1]
+    return a_new_seq
+
+
+def remove_every_other(a):
+    a_seq = a
+    a_new_seq = a_seq[0::2]
+    return a_new_seq
+
+
+def remove_f4_l4(a):
+    a_seq = a
+    a_new_seq = a_seq[4:-4]
+    return a_new_seq
+
+
+def reverse_seq(a):
+    a_seq = a
+    a_new_seq = a_seq[::-1]
+    return a_new_seq
+
+
+def reorder_seq(a):
+    a_seq = a
+    a_new_seq = a_seq[3:-3] + a_seq[-3:] + a_seq[0:3]
```

Comment:
here is the correct assert for this function:
`reorder_seq(2, 54, 13, 12, 5, 32) == (13, 12, 5, 32, 2, 54)`

when they refer to thirds they mean third part of that sequence which you can get by dividing the sequence into 3 parts.